crates/language-tests/tests/language/statements/define/field/empty_nested_arrays.surql

```diff
@@ -8,15 +8,15 @@ value = "NONE"
 value = "NONE"
 
 [[test.results]]
-value = "[{ id: person:test, settings: {  } }]"
+value = "[{ id: person:test, settings: { nested: [1, 2, 3, 4, 5] } }]"
```

Comment:
The point of this test is that the nested value is not preserved, it tests explicitely what happens without the `FLEXIBLE` keyword